File: src/main/scala/de/mineformers/allomancy/internal/network/EntityMetalBurnerUpdate.scala
package de.mineformers.allomancy.internal.network

import de.mineformers.allomancy.metal.{MetalBurner, MetalStorage}
import tessera.io.Message

/**
  * EntityMetalStorageUpdate
  *
  * @author PaleoCrafter
  */
case class EntityMetalBurnerUpdate(var entity: Int, var storage: MetalBurner) extends Message {
  def this() = this(-1, null)
}

File: src/main/java/de/mineformers/allomancy/network/messages/EntityMetalBurnerUpdate.java
package de.mineformers.allomancy.network.messages;

import de.mineformers.allomancy.metal.MetalBurner;
import de.mineformers.allomancy.network.Message;

/**
 * EntityMetalBurnerUpdate
 *
 * @author PaleoCrafter
 */
public class EntityMetalBurnerUpdate extends Message
{
    public int entity;
    public MetalBurner burner;

    public EntityMetalBurnerUpdate()
    {
    }

    public EntityMetalBurnerUpdate(int entity, MetalBurner burner)
    {
        this.entity = entity;
        this.burner = burner;
    }
}
